Cython/Compiler/PyrexTypes.py

```diff
@@ -4190,7 +4195,8 @@ def create_type_wrapper(self, env):
         env.use_utility_code(CythonUtilityCode.load(
             "EnumType", "CpdefEnums.pyx",
             context={"name": self.name,
-                     "items": tuple(self.values)},
+                     "items": tuple(self.values),
+                     "enum_doc": self.doc},
```

Comment:
Just to simplify the next change we make in this place:
```suggestion
                     "enum_doc": self.doc,
                     },
```